### PR TITLE
feat(IDPay): [IODPAY-79] Add instruments list check in IDPay initiative configuration

### PR DESCRIPTION
**Pull request by mastro993**
## Short description
This PR adds the instruments list check in the initiative configuration and the the ability to complete the configuration of an initiative without enrolling any instruments.

#### Example videos
End configuration without instruments | Add new instruments during configuration | Add instruments after initial configuration
:-: | :-: | :-:
<video src='https://user-images.githubusercontent.com/6160324/212027142-1c429e9d-64b1-42fd-aa94-7db734459182.mp4' width=100/> | <video src='https://user-images.githubusercontent.com/6160324/212027703-2e18e50c-edc1-4764-bec9-dc233b6a7620.mp4' width=100/> | <video src='https://user-images.githubusercontent.com/6160324/212036541-720c876f-9cb4-4c7b-b3bb-d7c220d11b21.mp4' width=100/>

## List of changes proposed in this pull request
* Added `areInstrumentsSkipped` in context to determine if instruments were skipped
* Added `EVALUATING_INSTRUMENTS` state which checks if user has payment methods.
* Added `SKIP` and `ADD_PAYMENT_METHOD` events to the configuration machine
* Added `navigateToAddPaymentMethodScreen` action
* `ConfigurationSuccessScreen` refactoring to display relevant info if configuration ended without instruments
* `InstrumentsEnrollmentScreen` refactoring:
    - Added new footer buttons if screen is displayed only for instruments configuration
    - Renamed `View`, `Text` and `List` components imported from `native-base` to `NBView`, `NBText` and `NBList`
    - Locales refactoring

## How to test
To test this PR you need to start the configuration of a new initiative and tap on the left button during the instrument enrollment phase.

### CODE REVIEW

File: ts/features/idpay/initiative/configuration/screens/InstrumentsEnrollmentScreen.tsx [hunk 1]
```diff
@@ -55,9 +58,17 @@ const InstrumentsEnrollmentScreen = () => {
     return instrumentItemsRef.current;
   };
 
-  const selectedInstrumentIdRef = useRef<number | undefined>(undefined);
+  const selectedInstrumentIdRef = React.useRef<number | undefined>(undefined);
 
   const isLoading = useSelector(configurationMachine, isLoadingSelector);
+  const initiativeDetails = useSelector(
```
Comment (hevelius) [inline ts/features/idpay/initiative/configuration/screens/InstrumentsEnrollmentScreen.tsx:64]: What do you think about using useIOSelector here?

File: ts/features/idpay/initiative/configuration/screens/ConfigurationSuccessScreen.tsx [hunk 2]
```diff
@@ -24,31 +34,73 @@ const ConfigurationSuccessScreen = () => {
   const { initiativeName } = initiativeDetails;
 
   const handleNavigateToInitiativePress = () => {
-    send({ type: "COMPLETE_CONFIGURATION" });
+    configurationMachine.send({ type: "COMPLETE_CONFIGURATION" });
+  };
+
+  const handleAddPaymentMethodButtonPress = () => {
```
Comment (hevelius) [inline ts/features/idpay/initiative/configuration/screens/ConfigurationSuccessScreen.tsx:40]: Maybe this braces {} could be removed.

File: ts/features/idpay/initiative/configuration/screens/InstrumentsEnrollmentScreen.tsx (hunk 1 above)
Comment (mastro993) [reply ts/features/idpay/initiative/configuration/screens/InstrumentsEnrollmentScreen.tsx:64]: `useIOSelector` is a typed selector for the redux store, in this case `useSelector` is exported from `@xstate/react` and it's used to get data from XState context :)

File: ts/features/idpay/initiative/configuration/screens/ConfigurationSuccessScreen.tsx (hunk 2 above)
Comment (mastro993) [reply ts/features/idpay/initiative/configuration/screens/ConfigurationSuccessScreen.tsx]: Done in https://github.com/pagopa/io-app/pull/4290/commits/54f91ba77a77c9b030a4f37e4b6575d1d80b77e1

**Review by francescopersico — APPROVED** [review]

**Pull request merged**